src/main/java/io/openbpm/control/service/job/impl/JobServiceImpl.java

```diff
@@ -171,4 +187,27 @@ protected List<JobQueryDtoSortingInner> createSortOptions(Sort sort) {
 
         return jobQueryDtoSortingInners;
     }
+
+    protected String fallbackGetStacktrace(String jobId) {
+        try {
+            RestTemplate restTemplate = new RestTemplate();
+            HttpHeaders headers = new HttpHeaders();
+            headers.setAccept(Collections.singletonList(MediaType.TEXT_PLAIN));
+            headers.setBasicAuth(camundaUsername, camundaPassword);
```

Comment:
Business process engines have different authentication methods.

Implement a custom query in the EngineRestClient class, see examples of queries in the class:
https://github.com/openbpm-platform/openbpm-control/blob/main/src/main/java/io/openbpm/control/service/client/EngineRestClient.java